src/components/wizard-navigation-bar.component.html

```diff
@@ -13,7 +13,11 @@
         <ng-container *ngIf="step.stepTitleTemplate" [ngTemplateOutlet]="step.stepTitleTemplate.templateRef"></ng-container>
         <ng-container *ngIf="!step.stepTitleTemplate">{{step.stepTitle}}</ng-container>
       </div>
-      <div class="step-indicator" [ngStyle]="{ 'font-family': step.navigationSymbol.fontFamily }">{{step.navigationSymbol.symbol}}</div>
+      <div *ngIf="step.stepSymbolTemplate" class="step-indicator">
+        <ng-container [ngTemplateOutlet]="step.stepSymbolTemplate.templateRef"></ng-container>
+      </div>
+      <div *ngIf="!step.stepSymbolTemplate" class="step-indicator"
```

Comment:
Is there a reason, why we don't implement this similar to the step title?
To clarify what I mean, I would expect something like this:
```html
<div class="step-indicator">
        <ng-container *ngIf="step.stepSymbolTemplate" [ngTemplateOutlet]="step.stepSymbolTemplate.templateRef"></ng-container>
        <ng-container *ngIf="!step.stepSymbolTemplate" [ngStyle]="{ 'font-family': step.navigationSymbol.fontFamily }">{{step.navigationSymbol.symbol}}</ng-container>
</div>
```
Is this possible?

Reply:
Ng-container is a 'virtual' Angular component that does not have a corresponding element in the DOM, so `[ngStyle]` cannot put `style=""` anywhere and it has no effect. Demo:
https://stackblitz.com/edit/angular5-template-oc4v8z?file=src/app/app.component.html

Reply:
Yes, you're right, thanks for reminding me :)
Is there a way to structure it a little bit better, I mean two of the four lines are almost the same...